### ISSUE TITLE
-traces should include tags

### ISSUE THREAD

**Issue by ahh** [google/pprof#152]
Attached a profile (zipped gzip is silly, but Github won't let me upload "heapz".)
```
pprof -raw heapz
...

Samples:
objects/count space/bytes[dflt]
       7438   60936269: 1 2 3 4 5 6
                bytes:[8192] request:[8191]
        238   62466710: 7 8 9 10 5 6
                bytes:[262144] request:[262143]
       3341   54739221: 7 8 9 10 5 6
                bytes:[16384] request:[16383]
       2064   67634517: 1 2 3 4 5 6
                bytes:[32768] request:[32767]
      29710   60847108: 1 2 3 4 5 6
                bytes:[2048] request:[2047]
        396   51920894: 1 2 3 4 5 6
                bytes:[131072] request:[131071]
      63503   65027585: 7 8 9 10 5 6
                bytes:[1024] request:[1023]
       1105   72423082: 7 8 9 10 5 6
                bytes:[65536] request:[65535]
        108   56884975: 11 12 13 3 4 5 6
                bytes:[524288] request:[524287]
      15887   65075220: 7 8 9 10 5 6
                bytes:[4096] request:[4095]
...
pprof -traces heapz
...

-----------+-------------------------------------------------------
   69.07MB   [UNKNOWN]
             [UNKNOWN]
             [UNKNOWN]
             [UNKNOWN]
             __libc_start_main
             [UNKNOWN]
-----------+-------------------------------------------------------
   54.25MB   [UNKNOWN]
             [UNKNOWN]
             [UNKNOWN]
             [UNKNOWN]
             [UNKNOWN]
             __libc_start_main
             [UNKNOWN]
-----------+-------------------------------------------------------
   62.06MB   [UNKNOWN]
             [UNKNOWN]
             [UNKNOWN]
             [UNKNOWN]
             __libc_start_main
             [UNKNOWN]
-----------+-------------------------------------------------------
...etc
```

the "traces" output (which is much more useful (as it can be filtered, etc)) should contain the bytes/request tags.
[heapz.zip](https://github.com/google/pprof/files/1131685/heapz.zip)

**Comment by ahh** [comment]
ping?

**Comment by aalexand** [comment]
@nolanmar511 could you look at it?

**Comment by aalexand** [comment]
Should be fixed by #179
